src/dependencies.jl

```diff
@@ -871,10 +1049,22 @@ macro install(_libmaps...)
                             end
                             """)
                         println(depsfile, "# Load dependencies")
+                        exec_deps = Dict()
                         for libkey in keys($libmaps)
-                            ((cached = get(load_cache,string(libkey),nothing)) === nothing) && continue
-                            println(depsfile, "@checked_lib ", $libmaps[libkey], " \"", escape_string(cached), "\"")
+                            cached = get(load_cache, string(libkey), nothing)
+                            if cached !== nothing
+                                (dep, path) = cached
+                                if isa(dep, BinDeps.LibraryDependency)
+                                    println(depsfile, "@checked_lib ", $libmaps[libkey], " \"", escape_string(path), "\"")
+                                else
```

Comment:
Maybe do `elseif isa(dep, BinDeps.ExecutableDependency)` for clarity, and add a final `else` branch raising an error.